### DISCUSSION TITLE
尊敬的老师您好，我们团队开发的 LaTeX 编辑器：LoongTeX

### DISCUSSION THREAD

**Discussion by loongtex** [ustctug/ustcthesis#412] [Q&A]
老师好：

我们团队开发了LaTeX 编辑器：LoongTeX，目前已经正式发布了，对于咱们国内用户来讲，是非常好用的，能否请求把我们的编辑器地址加在咱们项目的 README 文件中呢？感谢🙏

我们的平台地址：https://www.loongtex.com

**Comment by zepinglee** [comment]
已经添加。

另外我想问一下：

1. 如何更新 loongtex 网站上的模板？我刚刚发布了 v4.0.0-beta.4，而且最近在持续更新中。
2.  目前 LoongTeX 平台上的中文字体似乎只有 Fandol？

**Reply by loongtex** [reply]
感谢～

关于您提出的两个问题：

1. LoongTeX 的用户可以自己发布模板（这个功能我们刚刚上线），方便的话，可以加个微信，我把您的账号做一个“企事业单位官方认证”，这样您就可以在 LoongTeX 上自己发布自己的模板了。（我的微信：18301023060）
2. 目前 LoongTeX 平台上的中文字体可能还有欠缺，我们正在逐步的完善中。我试了一下咱们的模板是可以正常编译的，或许有些字体不太准确，这个咱们随时沟通，我来完善。

最后，献上最真诚的感谢～

以上；
